### PR TITLE
#161124006: Implement ability to view related articles

### PR DESCRIPTION
**Pull request by lauragift21**
#### What does this PR do?
It implements the ability to view related articles

#### Description of Task to be completed?
- Add related article functionality
- Write tests for functionality

#### How should this be manually tested?
- clone the repo using git clone {the repo url} from your terminal of choice
- cd into the directory and `npm install` and then run `npm start` 
- Navigate to `localhost:3000/articles/staying-fit` to view a sample of how the feature is implemented

#### What are the relevant pivotal tracker stories?
[#161124006](https://www.pivotaltracker.com/story/show/161124006)

#### Screenshots (if appropriate)

screen shot 2018-10-17 at 5 35 38 pm
screen shot 2018-10-17 at 5 35 57 pm

### CODE REVIEW

File: src/tests/actions/relatedArticle.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import * as types from "../../actionTypes/relatedArticle";
+
+const API = process.env.REACT_APP_API;
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const moxios = new MockAdapter(axios);
+
+describe("getRelatedArticle", () => {
+  xit("dispatches FETCH_RELATED_ARTICLE_SUCCESS", () => {
```
Comment (CEOehis) [inline src/tests/actions/relatedArticle.test.js]: This test is skipped.

**Review by CEOehis — CHANGES_REQUESTED** [review]

File: src/tests/actions/relatedArticle.test.js (hunk 1 above)
Comment (lauragift21) [reply src/tests/actions/relatedArticle.test.js]: Thanks I missed that

**Review by gonuoha — APPROVED** [review]
LGTM

**Review by CEOehis — APPROVED** [review]
👍 	LGTM

**Review by johnkingzy — APPROVED** [review]
LGTM

**Pull request merged**